src/NHSD.BuyingCatalogue.Identity.Api/Views/Shared/_CookieConsent.cshtml

```diff
@@ -0,0 +1,12 @@
+<section data-test-id="cookie-banner" class="bc-c-cookies nhsuk-header--white nhsuk-u-padding-top-3" aria-labelledby="cookies" role="banner">
+    <div class="nhsuk-width-container">
+        <h2 class="nhsuk-heading-m">Cookies on the Buying Catalogue</h2>
+        <p class="hsuk-body">We've put small files called cookies on your device to make the Buying Catalogue website work.</p>
+        <p class="hsuk-body">You can read more about our <a href="/privacy-policy">cookies and privacy policy</a>.</p>
```

Comment:
I think you'll need to inject `PublicBrowseSettings` and use `@PublicBrowseSettings.BaseAddress` here – see the Consent view, for example.

Reply:
Good point - added model for base URL to be passed through to partial as settings injected in the calling view.